### ISSUE TITLE
ERROR : : error reading source directory: directory not found

### ISSUE THREAD

**Issue by itcrowdsource** [cloudrkt/lxdbackup#2]
Hi,

I'm trying to use this script, the content seems very straightforward. But unfortunately I'm getting an error while transferring the image to an object storage. I'm able to copy the archive manually with the rclone copy command. But the script seems to be having problems handling the rclone command.
Does anyone know how to solve this issue?
```
root@<LXD host>:/tmp/lxdbackup# /scripts/lxdbackup <servername>
Info: Container <servername> found, continuing..
Snapshot: Succesfully created snaphot 06-25-18-14-51 on container <servername>
Container published with fingerprint: 149b982c7b801b4618f8d07649abd2c94f9bd51e7d5845c32a11d7f621ff994c
Publish: Succesfully published an image of <servername>-BACKUP-06-25-18-14-51 to <servername>-BACKUP-06-25-18-14-51-IMAGE
Image exported successfully!
Image: Succesfully exported an image of <servername>-BACKUP-06-25-18-14-51-IMAGE to /tmp/lxdbackup/<servername>-BACKUP-06-25-18-14-51-IMAGE.tar.gz
Target directory: Succesfully created the <remote>:lxdbackup directory
2018/06/25 14:53:12 DEBUG : rclone: Version "v1.42-005-g56e1e820β" starting with parameters ["/usr/bin/rclone" "-vv" "copy" "/tmp/lxdbackup/<servername>-BACKUP-06-25-18-14-51-IMAGE.tar.gz" "<remote>:lxdbackup/<servername>/"]
2018/06/25 14:53:12 DEBUG : Using config file from "/root/.config/rclone/rclone.conf"
2018/06/25 14:53:12 ERROR : : error reading source directory: directory not found
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for checks to finish
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for transfers to finish
2018/06/25 14:53:12 ERROR : Attempt 1/3 failed with 1 errors
2018/06/25 14:53:12 ERROR : : error reading source directory: directory not found
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for checks to finish
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for transfers to finish
2018/06/25 14:53:12 ERROR : Attempt 2/3 failed with 1 errors
2018/06/25 14:53:12 ERROR : : error reading source directory: directory not found
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for checks to finish
2018/06/25 14:53:12 INFO  : Swift container lxdbackup path <servername>/: Waiting for transfers to finish
2018/06/25 14:53:12 ERROR : Attempt 3/3 failed with 1 errors
2018/06/25 14:53:12 INFO  :
Transferred:      0 Bytes (0 Bytes/s)
Errors:                 1
Checks:                 0
Transferred:            0
Elapsed time:       900ms

2018/06/25 14:53:12 DEBUG : 6 go routines active
Could not create the <remote>:lxdbackup/<servername>/<servername>-BACKUP-06-25-18-14-51-IMAGE.tar.gz on <remote>:lxdbackup/<servername>/
Cleanup: Succesfully deleted snapshot <servername>/06-25-18-14-51 -
Fingerprint: 149b982c7b801b4618f8d07649abd2c94f9bd51e7d5845c32a11d7f621ff994c
Size: 561.12MB
Architecture: x86_64
Public: no
Timestamps:
    Uploaded: 2018/06/25 13:53 UTC
    Expires: never
    Last used: never
Properties:
Aliases:
    - <servername>-BACKUP-06-25-18-14-51-IMAGE
Cached: no
Auto update: disabled
Cleanup: Succesfully deleted copy <servername>-BACKUP-06-25-18-14-51-IMAGE
```

**Comment by dkruyt** [comment]
I got the same, found out that the backup file had a double extension. .tar.gz.tar.gz
So in the script it should be remove. 

So the following commit should be reverted to support 3.0

https://github.com/cloudrkt/lxdbackup/commit/11dec2ce046ca46328242429c791f2244de755df?_pjax=%23js-repo-pjax-container

**Comment by jeremybrown82** [comment]
Well are you going to revert it? This was opened a couple months back... It took some time to figure out that it wasn't working not because of me. It was at the point when I decided to post a bug report, then I found this thread. It would save others time if this was committed.

**Comment by oliverbienert** [comment]
I've just created a pull request on this issue.

**Comment by cloudrkt** [comment]
Fixed in 246487564d0a56b954a31e39bdebf27ad8057ee0
